lib/element.rb

```diff
@@ -111,6 +111,14 @@ def displayed?
     return false
   end
 
+  def not_displayed?
```

Comment:
I think I see where you are going with this. Correct me if I'm wrong, but you could add an `opt = {}` to `displayed?` and pass `timeout` to `element` on line `108`